portfolio/src/main/webapp/script.js

```diff
@@ -30,7 +30,35 @@ function findZoe() {
   const location = locations[Math.floor(Math.random() * locations.length)];
 
   // Add it to the page.
-  const link = document.getElementById("link");
+  const link = document.getElementById('link');
   link.innerText = location;
   link.href = locationToLink[location];
 }
+
+function grow() {
+  const srcToAlt = {
+    'images/basil1.jpg': 'Basil Day 0',
+    'images/basil2.jpg': 'Basil Day 10',
+    'images/basil3.jpg': 'Basil Day 20',
+    'images/basil4.jpg': 'Basil Day 30',
+    'images/basil5.jpg': 'Basil Day 40',
+    'images/basil6.jpg': 'Basil Day 60',
+  };
+  const image = document.getElementById('basil');
+  const endMsg = document.getElementById('endMsg');
+  console.log(`Let's grow some basil`);
+  endMsg.innerHTML = '';
```

Comment:
This element starts empty so it's not necessary to clear it.

Reply:
If someone clicks the button twice, the text box will have stored the message from the previous round of slideshow. By clearing the element at the start of every round, the message will only appear when the slide show stops at the last photo.

Reply:
got it. Can you add a comment to explain this line?

Reply:
Also, instead of innerHTML, use innerText.

Reply:
Both done :)